core/src/main/resources/hudson/logging/LogRecorder/configure.jelly

```diff
@@ -38,10 +38,10 @@ THE SOFTWARE.
     <p>
       ${%Enter the name of the logger you want to enable}.
     </p>
-    <div class="alert alert-warning">
-      ${%fine_warning}
-    </div>
     <f:form method="post" name="config" action="configSubmit">
+      <div class="alert alert-info jenkins-form-item">
```

Comment:
downgrade to info as it looks quite poor to have a page giving warnings on load, best to avoid the default being wrong imo (fine level with empty recorder).